skaffold.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: skaffold/v1
+kind: Config
+build:
+  artifacts:
+    - image: lab-partner
+      context: ./
+  local: {}
+
+deploy:
+  helm:
+    releases:
+      - name: lab-partner
+        chartPath: charts/lab-partner
+        wait: true
+        values:
+          image.name: lab-partner
+        setValues: 
+          ingress.host: ""
```

Comment:
I don't want to globally default this to anything, even an empty string.  let's remove `setValues` all together unless there is a good reason to keep it.